ontotext-yasgui-web-component/src/services/utils/html-util.ts

```diff
@@ -11,4 +11,24 @@ export class HtmlUtil {
     textArea.innerHTML = text;
     return textArea.value;
   }
+
+  static addScriptTag(url: string, onLoadHandler: () => void | undefined = undefined, async = false) {
+    const loader = document.createElement('script');
+    loader.setAttribute('src',url);
+    loader.async = async;
+    if (onLoadHandler) {
+      loader.addEventListener('load', onLoadHandler);
+    }
+    document.head.appendChild(loader);
+  }
+
+  static addCssTag(url: string, onLoadHandler: () => void | undefined = undefined, async = false) {
```

Comment:
What is this function supposed to do and why it's identical to `addScript`?

Reply:
This function have to load css. Fixed.